tests/client/core/test_utils.js

```diff
@@ -88,18 +88,19 @@ describe('getClientApplication', () => {
 
 
 describe('isValidClientApp', () => {
-  const fakeConfig = new Map();
-  fakeConfig.set('validClientApplications', ['firefox', 'android']);
+  // eslint-disable-next-line no-underscore-dangle
```

Comment:
I just added this rule in https://github.com/mozilla/addons-frontend/pull/819 If you rebase on master you'll get it.